Microsoft.PowerPlatform.Tooling.BatchedTelemetry Error: 2 : BatchedSender.SendTransmission Caught unhandled exception while trying to send a transmission.
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The remote name could not be resolved: 'westus2-2.in.applicationinsights.azure.com'
   at System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   at System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PowerPlatform.Tooling.BatchedTelemetry.BatchedUpload.BatchedSender.Transmitter.<SendAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.PowerPlatform.Tooling.BatchedTelemetry.BatchedUpload.BatchedSender.<SendTransmission>d__16.MoveNext()
Microsoft.PowerPlatform.Tooling.BatchedTelemetry Warning: 4 : BatchedSender.SendTransmission returned false after processing transmission file: C:\Users\igewebs\AppData\Local\Microsoft\PowerAppsCli\telemetry-cache\SolutionPackager\64288-638310018678526354.sending
Microsoft.PowerPlatform.Tooling.BatchedTelemetry Error: 2 : BatchedSender.SendTransmission Caught unhandled exception while trying to send a transmission.
System.Net.Http.HttpRequestException: An error occurred while sending the request. ---> System.Net.WebException: The remote name could not be resolved: 'westus2-2.in.applicationinsights.azure.com'
   at System.Net.HttpWebRequest.EndGetRequestStream(IAsyncResult asyncResult, TransportContext& context)
   at System.Net.Http.HttpClientHandler.GetRequestStreamCallback(IAsyncResult ar)
   --- End of inner exception stack trace ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PowerPlatform.Tooling.BatchedTelemetry.BatchedUpload.BatchedSender.Transmitter.<SendAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Microsoft.PowerPlatform.Tooling.BatchedTelemetry.BatchedUpload.BatchedSender.<SendTransmission>d__16.MoveNext()
Microsoft.PowerPlatform.Tooling.BatchedTelemetry Warning: 4 : BatchedSender.SendTransmission returned false after processing transmission file: C:\Users\igewebs\AppData\Local\Microsoft\PowerAppsCli\telemetry-cache\SolutionPackager\64288-638310018678526354.sending
